hw/application_fpga/fw/README.md

```diff
@@ -73,7 +73,9 @@ Dev Handbook for specific details.
 ## Memory constraints
 
 - ROM: 6 kByte.
-- FW\_RAM: 2 kByte.
+- FW\_RAM: 4 kByte.
```

Comment:
I added a firmware doc update to reflect the changes to FW_RAM. Feel free to squash.